### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by henrykgggg**
## Description
In this PR, we created the projects and about me sections.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, We:

- [x] used CSS `grid` and `flexbox`.
- [x] used `ul` and `li` by displaying lists.
- [x]  implemented `button` interaction.

 # Happy reviewing 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,35 +53,40 @@ <h4 class="icon">LET’S CONNECT</h4>
...
+      <div class="card-contain">
+        <ul class="cards-list flex_column">
+          <li class="hold-item flex-culomn">
+            <img
+              class="hold-image"
+              src="img/Snapshoot-Portfolio1.png"
+              alt="Portfolio Portfolio"
+            />
+            <div class="list-container">
+              <h2>Tonic</h2>
```
Comment (Kingstalux) [inline index.html:66]: - [ ] Great job so far 👍 Kindly make sure that the background image of each card is white throughout so as to match the Figma template. The sides of your cards do not have a white background.
- Your design:
Annotation 2022-03-26 195319
- Figma design:
Annotation 2022-03-26 195302

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @henrykgggg , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966

## Highlights
✅ Great pull request description 🔥 
✅ Design follows Figma 👏 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly fix the following broken image 👍 
- Your design:
Annotation 2022-03-26 195107

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (henrykgggg) [reply index.html]: Hello, @Kingstalux I have made the requested changes. Thank you for reviewing my PR. 🤝

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @henrykgggg,

Good job working on the changes asked by the previous reviewer. Keep rocking high 💪💪💪  

ClappingLeonardoDicaprioGIF

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**